server/dash/pages/nc_summary_comparison.py

```diff
@@ -159,14 +163,56 @@ def generate_comparison_filtered_df(api_data_df, selected_nc):
         selected_nc: A string argument automatically detected by Dash callback
          function when "nc_comp_dropdown" element is selected in the layout.
     Returns: 
-        Pandas dataframe with requests from the nc selected by nc_comp_dropdown.
+        df: Pandas dataframe with requests from the nc selected by nc_comp_dropdown.
+        create_dt_col_name: String column name of the new datetime column.
     """
     if not selected_nc:
         df = api_data_df
     else:
         df = api_data_df[api_data_df["councilName"] == selected_nc]
-    df = add_datetime_column(df, "createdDate")
-    return df
+    df, create_dt_col_name = add_datetime_column(df, "createdDate")
+    return df, create_dt_col_name
+
+def generate_summary_statistics(nc_comp_dropdown):
+    """Generates the summary statistics for neighborhood council
+    comparisons.
+
+    Args:
+        nc_comp_dropdown: name of the selected neighborhood council.
+
+    Returns:
+        total_req_card: integer for the the total number of request 
+            in first selected neigborhood council.
+        num_days_card: integer for the total number of days the data 
+            available in first selected neighborhood council span.
+    """
+    df_nc1, create_dt_col_name = generate_comparison_filtered_df(api_data_df, nc_comp_dropdown)
+    total_req_card = df_nc1.shape[0]
+    num_days_card = np.max(df_nc1[create_dt_col_name].dt.day) - \
+                           np.min(df_nc1[create_dt_col_name].dt.day) + 1
+    return total_req_card, num_days_card
+
+def req_source_helper(nc_comp_dropdown):
```

Comment:
Let's rename to something like 'get_request_source_bar_chart'

Reply:
Thanks for the suggestion - resolved.